src/main/java/seedu/club/logic/commands/AddPollCommand.java

```diff
@@ -50,6 +51,11 @@ public AddPollCommand(Poll poll) {
     public CommandResult executeUndoableCommand() throws CommandException {
         requireNonNull(model);
         try {
+            if (requireToSignUp()) {
```

Comment:
Maybe, for v1.5rc, we can do this check in LogicManager before executing the command? This way, you won't have to repeat the same code for every command.

Reply:
but not all commands requrie to login like exit, signup.

Reply:
Good point.We can override the requireLogIn and requireSignUp in those methods to always return false